src/nCr/index.js

```diff
@@ -0,0 +1,36 @@
+/* nCr
+ * Function: nCr()
+ */
+
+const validate = require('../validation/non-negative-integer');
+
+/**
+ *
+ * @param {number, number} number
+ * @returns {number}
+ */
+
+function nCr(number1, number2) {
+
+	if(number1 < number2){
+		return "n >= r condition not satisfied.";
+	}
+
+	let res = 1;
+	let i = 0;
+
+  if ( number2 > number1 - number2 )
+   	number2 = number1 - number2;
+
+  for (; i < number2; ++i) {
```

Comment:
Can you please use the factorial function already defined in the Mathball library, rather than implementing one yourself?
This should reduce the redundancy in the code base, ensuring cleaner code.

Reply:
Calculating factorial for n, and then for r, and hence dividing them is not efficient method. It will often lead to very large number. That's why I have not used factorial function.
Here, the code performs division, while calculating factorial n, and hence this improves the performance.

Reply:
Yes, this makes more sense while implementing for larger numbers, my bad.
This part of the code looks good to go.